.gitlab-ci.yml

```diff
@@ -55,15 +55,28 @@ variables:
   RUSTY_CACHIER_COMPRESSION_METHOD: zstd
   ZOMBIENET_IMAGE:                 "docker.io/paritytech/zombienet:v1.3.22"
 
-default:
-  retry:
-    max: 2
-    when:
-      - runner_system_failure
-      - unknown_failure
-      - api_failure
-  interruptible:                   true
-  cache:                           {}
+.shared-default: &shared-default
+    retry:
+      max: 2
+      when:
+        - runner_system_failure
+        - unknown_failure
+        - api_failure
+    cache: {}
+
+.default-pipeline-definitions:
+  default:
+    <<: *shared-default
+    interruptible: true
+
+.crate-publishing-pipeline-definitions:
+  default:
+    <<: *shared-default
+    # The crate-publishing pipeline should not be interruptible so that we'll be able to run the
+    # publishing automation jobs to completion despite the "Auto-cancel redundant pipelines" CI
+    # setting (relevant because the crate-publishing pipeline runs on `master`, so future pipelines
+    # on `master` might cancel it before the publishing jobs run to completion).
+    interruptible: false
```

Comment:
Please fix indent

Reply:
For some reason `.shared-default` wasn't working before with an indent of 2 spaces. I was getting the error `did not find expected key while parsing a block mapping at line`. Now it seems to work somehow. Fixed.